apps/mobile/src/components/TabNavivation/TabBar.tsx

```diff
@@ -1,14 +1,37 @@
 import type { FC } from 'react';
-import type { ViewStyle } from 'react-native';
+import type { ImageSourcePropType, ViewStyle } from 'react-native';
 import { StyleSheet, View } from 'react-native';
 import type { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import { useDrawerStatus } from '@react-navigation/drawer';
+import type { DrawerNavigationHelpers } from '@react-navigation/drawer/lib/typescript/src/types';
```

Comment:
don't import from this kind of path

Reply:
the `DrawerNavigationHelpers` is exposed in this lib path, might I re-export it in another the file (such as `@walless/abc`) and import from that file for a prettier import (`import type { DrawerNavigationHelpers } from '@walless/abc'`)